code/style.css

```diff
@@ -1,5 +1,156 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
+img {
+  font-size: 14px;
+}
 body {
-  background-color: lightblue;
+  font-family: "Helvetica Neue", Arial, Helvetica, sans-serif;
+  margin: 100px 0px;
+  background-color: rgb(252, 253, 254);
+}
+h1 {
+  display: left;
```

Comment:
Here you can read about what values you can give the display-property: https://www.w3schools.com/cssref/pr_class_display.asp